src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -70,7 +70,7 @@ public function getStream()
      */
     protected function doWrite($message, $newline)
     {
-        if (false === @fwrite($this->stream, $message.($newline ? PHP_EOL : ''))) {
+        if (false === @fwrite($this->stream, $message) || ($newline && (false === @fwrite($this->stream, PHP_EOL)))) {
```

Comment:
In this case no Exception would be thrown if something goes wrong with writing PHP_EOL.